src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function Calculator() {
```

Comment:
- [ ] Good job adding the calculator component 👍🏽 . Please ensure the `Calculator.js` component is written in a class-based component as specified in the project requirement 👍🏽.